mock/Cargo.toml

```diff
@@ -23,7 +23,7 @@ optional = true
 version = "0.0"
 
 [dependencies.rusoto_core]
-version = "0.25.0"
+version = "> 0.25.0"
```

Comment:
I kind of wonder if `mock` should just be put back into `core`? I didn't realize it had to be published as well, and now it's just another crate to maintain for not really any gain.